From 1902d0e45a1c479e717e13fefd5625b05b966cf6 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:23:38 -0400
Subject: [cpufreq] Remove cpufreq_generic_exit()

Message-id: <1409235879-1283-125-git-send-email-prarit@redhat.com>
Patchwork-id: 89460
O-Subject: [RHEL7.1 PATCH BZ 1134639 124/185] cpufreq: Remove cpufreq_generic_exit()
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 979d86fac5e6dbf5c939e9a2a9fe294c9e05e42f
Author: Viresh Kumar <viresh.kumar@linaro.org>
Date:   Mon Mar 10 14:53:34 2014 +0530

    cpufreq: Remove cpufreq_generic_exit()

    cpufreq_generic_exit() is empty now and can be deleted.

    Signed-off-by: Viresh Kumar <viresh.kumar@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/p4-clockmod.c b/drivers/cpufreq/p4-clockmod.c
index 3d1cba9..74f593e 100644
--- a/drivers/cpufreq/p4-clockmod.c
+++ b/drivers/cpufreq/p4-clockmod.c
@@ -237,7 +237,6 @@ static struct cpufreq_driver p4clockmod_driver = {
  .verify  = cpufreq_generic_frequency_table_verify,
  .target_index = cpufreq_p4_target,
  .init  = cpufreq_p4_cpu_init,
- .exit  = cpufreq_generic_exit,
  .get  = cpufreq_p4_get,
  .name  = "p4-clockmod",
  .attr  = cpufreq_generic_attr,
diff --git a/include/linux/cpufreq.h b/include/linux/cpufreq.h
index 66f22a6..a6a48e4 100644
--- a/include/linux/cpufreq.h
+++ b/include/linux/cpufreq.h
@@ -499,11 +499,6 @@ unsigned int cpufreq_generic_get(unsigned int cpu);
 int cpufreq_generic_init(struct cpufreq_policy *policy,
   struct cpufreq_frequency_table *table,
   unsigned int transition_latency);
-static inline int cpufreq_generic_exit(struct cpufreq_policy *policy)
-{
- return 0;
-}
-
 ssize_t cpufreq_show_cpus(const struct cpumask *mask, char *buf);
 
 #ifdef CONFIG_CPU_FREQ
-- 
1.7.1